### PR TITLE
Changes for the potassium modeling.

### PR DESCRIPTION
**Pull request by dkeller9**

### CODE REVIEW

File: common/mod/ngv/GluReceive.mod
Comment (dkeller9) [inline common/mod/ngv/GluReceive.mod:1]: verbatim print statements removed

File: common/mod/ngv/vascouplingB.mod
Comment (dkeller9) [inline common/mod/ngv/vascouplingB.mod:1]: Changes made to the potassium-induced voltage-dependent pathway to allow the signal to propagate to the radius

Comment (dkeller9) [reply common/mod/ngv/vascouplingB.mod:1]: I subtracted init0 in order to keep the radius constant under steady-state conditions.

File: common/mod/ngv/vascouplingB.mod [hunk 1]
```diff
@@ -180,14 +181,16 @@ BREAKPOINT{
...
+:RESTORE NEXT
     kp = kp + dt*((1/VRpa)*j_BK*0.0001+(1/VRps)*(j_KIR)-Rdecay*1000*(kp-kpmin))
     :kp = kp + dt*((1/VRpa)*j_BK*0.0001-Rdecay*1000*(kp-kpmin))
 
     : Membrane potential of SMC cell. Added mechanism to come back to resting potential, otherwise stays hyperpolarized
     : Via activation of KIR channel
     :Vi = Vi + dt*(1/Cmu)*(I_KIR)*pow(10,7) - dt*keet*5*pow(10,-2)*(Vi+30): with inactivation
     :Vi = Vi + dt*(1/Cmu)*(I_KIR)*pow(10,7)  - dt*keet*5*pow(10,-2)*(Vi+30): with inactivation
-    Vi = Vi + dt*(1/Cmu)*(I_KIR)*pow(10,7)  - dt*eete*1.2-0.001*(Vi+30)
+    Vi = Vi + dt*(1/Cmu)*(I_KIR)*pow(10,7)  - 0*dt*eete*1.2-0.001*(Vi+30)
```
Comment (WeinaJi) [inline common/mod/ngv/vascouplingB.mod:193]: Hello @dkeller9 , is it times 0 `0*dt*eete...` on purpose ?

Comment (dkeller9) [reply common/mod/ngv/vascouplingB.mod:193]: Yes, this pathway was causing issues so I disabled it.

**Comment by WeinaJi** [comment]
Approved. After merging this PR, I will need to update the reference reports in the neurodamus CI tests. 
And to deploy the new mod files, a new tag of this repo is required. Please let me know when you expect them to be released.

**Comment by WeinaJi** [comment]
> What is the process of making a new tag for the release? I updated ngv.py in neurodamus and need to refer to these mod files to pass CI.

I could take care of the tagging and releasing. Would you like to release soon?
@tristan0x told me that he will validate the new mod files with the CI offline. @tristan0x , do they work?

**Comment by dkeller9** [comment]
Great thanks. Yes, it would be good to release now.

**Pull request merged**